src/app/application.cpp

```diff
@@ -269,11 +273,16 @@ void Application::processMessage(const QString &message)
         m_paramsQueue.append(params);
 }
 
-void Application::runExternalProgram(BitTorrent::TorrentHandle *const torrent) const
+void Application::runExternalProgram(const BitTorrent::TorrentHandle *torrent) const
 {
     QString program = Preferences::instance()->getAutoRunProgram();
     program.replace("%N", torrent->name());
     program.replace("%L", torrent->category());
+
+    QStringList tags = torrent->tags().toList();
+    std::sort(tags.begin(), tags.end(), Utils::String::naturalLessThan<Qt::CaseInsensitive>);
+    program.replace("%G", tags.join(','));
```

Comment:
IMO, it's time to convert these placeholders to more informative form, e.g.: %hash, %tags, %save_path, etc.

Reply:
>IMO, it's time to convert these placeholders to more informative form, e.g.: %hash, %tags, %save_path, etc.

It was made to be compatible with utorrent, it can be a long term goal, but not short term IMO.